docs/modules/administration/pages/installation.adoc

```diff
@@ -159,8 +162,10 @@ make install-sonarqube
 ----
 
 This will launch an instance of SonarQube.
-The script will prompt for a new admin password, set it and create an auth token to be used by the Jenkins pipelines.
-Both values can be written into `ods-configuration/ods-core.env`, which you then need to commit and push before continuing.
+The script will prompt for a new admin password, once set, an auth token that is used by the Jenkins pipelines will be generated.
+Both values are automatically written into `ods-configuration/ods-core.env`, which you then need to commit and push after adding the Bitbucket `ods-configuration` repo as the remote origin.
+
+Also, you have to add the cd-user as a user in the project permissions of your Bitbucket OpenDevStack project. Read permission is sufficient.
```

Comment:
This should be closer to the `prepare-bitbucket-repos` target, no?

Reply:
removed the remote origin part of the sentence!